entrypoint.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -e
 
+# collect static files
+python manage.py collectstatic --no-input
+
```

Comment:
this is because we have some HTML/CSS in the code now?

Reply:
I needed this when I was trying to add images. Although I discarded using images in the repo, this line should always be there. It was missed by Vaibhav during the setup.